.github/workflows/ci.yml

```diff
@@ -0,0 +1,271 @@
+name: mattermost-build
+on:
+  push
+defaults:
+  run:
+    shell: bash
```

Comment:
This can be omitted since bash is the default shell for Github Actions running on linux.